extensions/jdbc/jdbc-mysql/deployment/src/main/java/io/quarkus/jdbc/mysql/deployment/MySQLJDBCReflections.java

```diff
@@ -42,14 +42,19 @@ void registerDriverForReflection(BuildProducer<ReflectiveClassBuildItem> reflect
         reflectiveClass.produce(new ReflectiveClassBuildItem(false, false, com.mysql.cj.jdbc.Driver.class.getName()));
         reflectiveClass.produce(
                 new ReflectiveClassBuildItem(false, false, com.mysql.cj.conf.url.FailoverConnectionUrl.class.getName()));
+        new ReflectiveClassBuildItem(false, false, com.mysql.cj.conf.url.FailoverDnsSrvConnectionUrl.class.getName());
         reflectiveClass
                 .produce(new ReflectiveClassBuildItem(false, false, com.mysql.cj.conf.url.SingleConnectionUrl.class.getName()));
         reflectiveClass.produce(
-                new ReflectiveClassBuildItem(false, false, com.mysql.cj.conf.url.LoadbalanceConnectionUrl.class.getName()));
+                new ReflectiveClassBuildItem(false, false, com.mysql.cj.conf.url.LoadBalanceConnectionUrl.class.getName()));
+        new ReflectiveClassBuildItem(false, false, com.mysql.cj.conf.url.LoadBalanceDnsSrvConnectionUrl.class.getName());
```

Comment:
Indentation is incorrect here.

Reply:
Good catch, the reflective item was only instantiated and not registered.